erpnext/agriculture/doctype/destroyed_plant_log/destroyed_plant_log.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from frappe.model.document import Document
+
+class DestroyedPlantLog(Document):
+	def on_submit(self):
```

Comment:
Are we creating this document manually? I thought this was supposed to automatically get created whenever you destroy something in the Plant Batch document? @neilLasrado

Reply:
ya but I asked @neilLasrado he said we can create this manually also.